.vib/moodle/cypress/cypress/integration/moodle_spec.js

```diff
@@ -0,0 +1,26 @@
+/// <reference types="cypress" />
+
+it('adds new course', () => {
+  cy.login();
+  cy.contains('a', 'Site administration').click();
+  cy.contains('a', 'Courses').click();
+  cy.contains('a', 'Add a new course').click();
```

Comment:
When possible, you should use `cy.visit()` to directly access the "real" starting page where the test will take place. In this case, you may replace this code with `cy.visit('course/edit.php')` or something similar.